dev/doc/emacs.md

```diff
@@ -0,0 +1,233 @@
+# Emacs configuration for Coq development
```

Comment:
Isn't this stuff from the wiki?

Reply:
Yes, which is why I want to remove the wiki page (there's a comment at the bottom of the file that says `delete wiki page once merged`). This was a general sweep to consolidate information.

Reply:
Somehow I like it better on the wiki. Do we have any guidelines for deciding what goes in the wiki and what goes in the repo?

Reply:
None that I know.

If you like it better on the wiki, then I propose we move whatever I've put into `emacs.md` and `vim.md` into the wiki.

I wanted it in the repo so it is easier to link to (and to grep for)

Reply:
I think I agree with putting the editor configs on the wiki rather than on the repo since it's likely to change independently of Coq.